cylc/flow/platform_lookup.py

```diff
@@ -62,13 +62,13 @@ def forward_lookup(platforms, job_platform):
     ... }
     >>> job_platform = 'desktop22'
     >>> forward_lookup(platforms, job_platform)
-    'desktop22'
+    ('desktop22', 'desktop[0-9][0-9]|laptop[0-9][0-9]')
```

Comment:
I would use `\d` rather than `[0-9]` ... easier to read?  (however, no biggie).

Reply:
So would Mel and I, but there are deprecation warnings:
```
DeprecationWarning: invalid escape sequence \d
```
This should be fixed by forcing strings to be raw - but I think that should be a different issue.

Reply:
As this is a unit test not an actual configuration it doesn't really matter but anyway the fix is trivial:

```python
('desktop22', r'desktop\d\d|laptop\d\d)
```